lua/oil/mutator/parser.lua

```diff
@@ -203,10 +207,10 @@ M.parse = function(bufnr)
           end_lnum = i,
           col = 0,
         })
-        goto continue
+        return
```

Comment:
This doesn't seem right. Since the function wraps the for-loop I believe the `return` will be functioning like a `break`, not a `continue`